features/scv/jceks-impl/src/main/resources/META-INF/opennms/applicationContext-jceks-scv.xml

```diff
@@ -0,0 +1,15 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<beans xmlns="http://www.springframework.org/schema/beans"
+       xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+       xmlns:onmsgi="http://xmlns.opennms.org/xsd/spring/onms-osgi"
+       xsi:schemaLocation="http://www.springframework.org/schema/beans http://www.springframework.org/schema/beans/spring-beans-4.2.xsd
+  http://xmlns.opennms.org/xsd/spring/onms-osgi http://xmlns.opennms.org/xsd/spring/onms-osgi.xsd">
+
+    <bean name="jceksSecureCredentialsVault" class="org.opennms.features.scv.jceks.JCEKSSecureCredentialsVault">
+        <constructor-arg value="${opennms.home}/etc/scv.jce"/>
+        <constructor-arg value="${org.opennms.features.scv.jceks.key:QqSezYvBtk2gzrdpggMHvt5fJGWCdkRw}"/>
+    </bean>
+
+    <onmsgi:service interface="org.opennms.features.scv.api.SecureCredentialsVault" ref="jceksSecureCredentialsVault"/>
```

Comment:
this exports a second instance to the container, right? can we just alter the blueprint to consume this instance?

Reply:
Since LocationAware clients are only available on Spring, we don't use the blueprint for scv, we only have spring bean and export the same as osgi service.